rotkehlchen/chain/aggregator.py

```diff
@@ -921,16 +921,11 @@ def query_evm_chain_balances(self, chain: SUPPORTED_EVM_CHAINS) -> None:
         manager = cast('EvmManager', self.get_chain_manager(chain))
         native_token_usd_price = Inquirer().find_usd_price(manager.node_inquirer.native_token)
         chain_balances = self.balances.get(chain)
-        queried_balances = manager.node_inquirer.get_multi_balance(accounts)
-        for account, balance in queried_balances.items():
-            if balance == ZERO:
-                continue
-
-            usd_value = balance * native_token_usd_price
+        for account, balance in manager.node_inquirer.get_multi_balance(accounts).items():
             chain_balances[account] = BalanceSheet(
                 assets=defaultdict(Balance, {
-                    manager.node_inquirer.native_token: Balance(balance, usd_value),
-                }),
+                    manager.node_inquirer.native_token: Balance(balance, balance * native_token_usd_price),  # noqa: E501
```

Comment:
I don't understand the fix. This needs a comment explaining why you added the `if balance != ZERO else {}`. How does that affect the specific case of the user with stETH makerDAO vault?